js/apps/admin-ui/src/realm-settings/OpenIdEndpointConigurationTab.tsx

```diff
@@ -0,0 +1,175 @@
+import type RealmRepresentation from "@keycloak/keycloak-admin-client/lib/defs/realmRepresentation";
+import {
+  ActionGroup,
+  Button,
+  FormGroup,
+  PageSection,
+  InputGroup,
+  TextInput,
+} from "@patternfly/react-core";
+import { useEffect, useState } from "react";
+import { Controller, useForm } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import { FormAccess } from "../components/form/FormAccess";
+import { HelpItem } from "ui-shared";
+import { addTrailingSlash } from "../util";
+import { adminClient } from "../admin-client";
+import { MinusIcon, PlusIcon } from "@patternfly/react-icons";
+import { getAuthorizationHeaders } from "../utils/getAuthorizationHeaders";
+
+type RealmSettingsThemesTabProps = {
+  realm: RealmRepresentation;
+  save: (realm: RealmRepresentation) => void;
+};
+
+export const OpenIdEndpointConigurationTab = ({
+  realm,
+  save,
+}: RealmSettingsThemesTabProps) => {
+  const { t } = useTranslation();
+
+  const [newClaim, setNewClaim] = useState<string>("");
+  const { control, handleSubmit, setValue } = useForm<RealmRepresentation>();
+
+  const setupForm = () => {
+    let attributes = realm.attributes || {};
+    
+    if (!("claimsSupported" in attributes)) {
+      getOpenIdEndpointConfiguration().then((result) => {
+        attributes.claimsSupported = result.claims_supported.join(",");
+        setValue("attributes", attributes);
+      });
+    }
+    else{
+      setValue("attributes", attributes);
+    }
+    
+  };
+
+  useEffect(setupForm, []);
+
+  async function getOpenIdEndpointConfiguration(): Promise<any> {
+    const response = await fetch(
+      `${addTrailingSlash(adminClient.baseUrl)}/realms/${
```

Comment:
Rather than joining the URL here manually, use the [`joinPath()`](https://github.com/keycloak/keycloak/blob/4bcc5df8585257c2b6282d421f142e27b3a38443/js/apps/admin-ui/src/utils/joinPath.ts#L3) utility instead.

Reply:
Removed as mentioned above.